protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -1189,7 +1177,7 @@ message ListTransfersRequest {
     // Public key for which to request transfers
     optional string pubkey = 1;
     // Transfer directions (sender, receiver) to include in the response data
-    TransferDirection direction = 2;
+    TransferDirection direction = 2 [(google.api.field_behavior) = REQUIRED];
```

Comment:
Not sure if this one is quite true at the moment. If I don't supply a public key I can omit the transfer direction:
```
https://api.n06.testnet.vega.xyz/api/v2/transfers
```

I found this API a bit weird actually in that if the above list all transfers it feels like:
```
https://api.n06.testnet.vega.xyz/api/v2/transfers?partyId=BLAH
```
would list all transfers involving `BLAH` but it doesn't because then we care for the direction and `UNSPECIFIED` makes it an error. But then also having `UNSPECIFIED => TRANSFER_TO_OR_FROM` looks a bit weird from the proto's point of view. Maybe we just need better doc-strings.

I don't know, but anyway for now I don't think it should be marked as required?

Reply:
You're right, I saw the check for the direction, but missed the fact it is only checked if a party ID is provided. I will update the doc-string to provide the information accordingly.

Reply:
Addressed https://github.com/vegaprotocol/vega/pull/8070/commits/16a86a18b2d74fe341c6e3d4e19eb7563618cc95